app/auth/GatehouseUserInfoResponseParser.scala

```diff
@@ -0,0 +1,35 @@
+package auth
+
+import com.gu.identity.auth.UserInfoResponseParser.{identityId, primaryEmailAddress}
+import com.gu.identity.auth.{CirceJsonParser, IdentityClaims, JsonString, UserInfoResponseParser, ValidationError}
+
+case class GatehouseIdentityClaims(
+    primaryEmailAddress: String,
+    identityId: String,
+    oktaId: String
+) extends IdentityClaims
+object GatehouseUserInfoResponseParser {
+  val OKTA_ID_CLAIM_NAME = "sub"
+
+  implicit val gatehouseUserInfoResponseParser: UserInfoResponseParser[GatehouseIdentityClaims] =
+    rawClaims =>
+      for {
+        emailAddress <- primaryEmailAddress(rawClaims)
+        id <- identityId(rawClaims)
+        oktaUid <- claim(OKTA_ID_CLAIM_NAME, rawClaims)
+
+      } yield GatehouseIdentityClaims(
+        primaryEmailAddress = emailAddress,
+        identityId = id,
+        oktaId = oktaUid
+      )
+  // TODO copied from the identityLibrary because it is private
+  private def claim(claimName: String, rawClaims: JsonString): Either[ValidationError, String] = {
+    val json = CirceJsonParser.toJson(rawClaims)
```

Comment:
Might be better to use play-json library to handle Json as we're using it elsewhere and it's part of the Play framework.  In fact, I'd be inclined to explicitly exclude `Circe` from the dependencies so that it isn't possible to use different Json handlers.